packages/types/src/SignedBlock.spec.js

```diff
@@ -2,11 +2,20 @@
 // This software may be modified and distributed under the terms
 // of the ISC license. See the LICENSE file for details.
 
+import extrinsics from '@polkadot/extrinsics/static';
+
+import Method from './Method';
 import rpc from './SignedBlock.rpc.json';
 import SignedBlock from './SignedBlock';
 
 describe('SignedBlock', () => {
-  const block = new SignedBlock().fromJSON(rpc.result);
+  let block;
```

Comment:
This really makes a difference?

Reply:
It is a bit more inefficient. Just wanted to keep it together in beforeEach